pep-0012.rst

```diff
@@ -50,44 +50,36 @@ To use this template you must first decide whether your PEP is going
 to be an Informational or Standards Track PEP.  Most PEPs are
 Standards Track because they propose a new feature for the Python
 language or standard library.  When in doubt, read PEP 1 for details
-or contact the PEP editors <peps@python.org>.
+or contact a PEP editor for assistance.
```

Comment:
Would it be possible to provide a link to the list of PEP editors?

Reply:
Hmm. PEP 1 just has this now: https://www.python.org/dev/peps/pep-0001/#pep-editors

Maybe we should advise people to open an issue on the repo?